GameData/ProbesBeforeCrew/Mod Support/ZsRestockPatch.cfg

```
@@ -0,0 +1,50 @@
+// Zee's Probes Before Crew Restock Patch
+// Version 2.10.0
+
+/////////// Restock Texture Support for Custom Parts
+
+
+
+@PART[size0_basicFin]:HAS[~RestockIgnore[*]]:FOR[000_Restock]

```

```suggestion
@PART[size0_basicFin]:HAS[~RestockIgnore[*]]:FOR[ProbesBeforeCrew]
```
Related to the other comments I wrote, should be the only instance in these files :D